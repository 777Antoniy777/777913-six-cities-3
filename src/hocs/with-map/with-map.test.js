import React from 'react';
import renderer from "react-test-renderer";
import withMap from "./with-map";

// set mocha data
const map = React.createRef();
const props = [];

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MapWrapperHoc = withMap(MockComponent);

it(`render withMap`, () => {
  const tree = renderer.create((
    <MapWrapperHoc
      props={props}
      map={map}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
