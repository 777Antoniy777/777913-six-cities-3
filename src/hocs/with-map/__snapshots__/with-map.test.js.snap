// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withMap 1`] = `<div />`;
